packages/screenshots/Classes/Configuration/Configuration.php

```diff
@@ -221,7 +221,34 @@ public function createBasicConfig(): void
                 ],
                 'Examples' => [
                     'screenshots' => [
-                        [
+                        "_default"=> [
+                            ["action"=> "resizeWindow", "width"=> 640,"height"=> 640],
+                            ["action"=> "setScreenshotsDocumentationPath", "path"=> "Documentation"],
+                            ["action"=> "setScreenshotsImagePath","path"=> "Images/AutomaticScreenshots"],
+                            ["action"=> "setScreenshotsRstPath","path"=> "Images/AutomaticScreenshots"],
+                            ["action"=> "setCodeSnippetsSourcePath","path"=> "typo3conf/ext/examples"],
+                            ["action"=> "setCodeSnippetsTargetPath", "path"=> "CodeSnippets"]
+                        ],
+                        "frontend" => [
+                            ["include" => "_default"],
+                            [
```

Comment:
Keep one action per one line.

Reply:
actually if you would use auto formating of the php file with our current .editorconfig all lines would look like mine not like yours.... I had my lines automatically formated